File: Task-03/C++/.cpp
#include <iostream>
#include <vector>

using namespace std;

// Function to check if a number is prime
bool isPrime(int num) {
    if (num <= 1) {
        return false;
    }
    for (int i = 2; i * i <= num; i++) {
        if (num % i == 0) {
            return false;
        }
    }
    return true;
}

int main() {
    int n;

    cout << "Enter a number (n): ";
    cin >> n;

    if (n < 2) {
        cout << "Prime numbers start from 2." << endl;
    } else {
        cout << "Prime numbers up to " << n << ":" << endl;
        for (int i = 2; i <= n; i++) {
            if (isPrime(i)) {
                cout << i << endl;
            }
        }
    }

    return 0;
}

File: Task-03/Go/.go
package main

import (
	"fmt"
	"math"
)

// Function to check if a number is prime
func isPrime(num int) bool {
	if num <= 1 {
		return false
	}
	for i := 2; i <= int(math.Sqrt(float64(num))); i++ {
		if num%i == 0 {
			return false
		}
	}
	return true
}

func main() {
	var n int
	fmt.Print("Enter a number (n): ")
	_, err := fmt.Scanf("%d", &n)

	if err != nil || n < 2 {
		fmt.Println("Prime numbers start from 2.")
	} else {
		fmt.Printf("Prime numbers up to %d:\n", n)
		for i := 2; i <= n; i++ {
			if isPrime(i) {
				fmt.Println(i)
			}
		}
	}
}
